spec/models/task_spec.rb

```diff
@@ -1,5 +1,17 @@
 require 'rails_helper'
 
-RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'タスク管理機能', type: :model do
+  it 'titleが空ならバリデーションが通らない' do
+    task = Task.new(theme: '', content: '失敗テスト')
+    expect(task).not_to be_valid
```

Comment:
好みもありますが下記のようにも記載できます！
```suggestion
    expect(task).to be_invalid
```

Reply:
ありがとうございます。ご記載いただいた方法の方がこなれている感じがしますので、参考にさせていただきます。